lambdas/ec2/schema_enforcer.py

```diff
@@ -0,0 +1,198 @@
+from __future__ import print_function
+
+import json
```

Comment:
do you actually use the `json` module in any of this code? Is there a reason to be including modules you're not using?